Orange/widgets/utils/__init__.py

```diff
@@ -90,3 +91,24 @@ def apply_all(seq, op):
     """Apply `op` on all elements of `seq`."""
     # from itertools recipes `consume`
     deque(map(op, seq), maxlen=0)
+
+
+def instance_tooltip(domain, row, skip_attrs=()):
+    def show_part(_point_data, singular, plural, max_shown, _vars):
+        cols = [escape('{} = {}'.format(var.name, _point_data[var]))
+                for var in _vars[:max_shown + 2]
```

Comment:
I suppose `+ 2` should become `+ len(skip_attrs)`. The code is copied from a function which skips at most two attributes (x and y).

Reply:
True.